github-upload-public-key/run.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -e
+
+CODER_ACCESS_URL="${CODER_ACCESS_URL}"
+CODER_OWNER_SESSION_TOKEN="${CODER_OWNER_SESSION_TOKEN}"
+CODER_EXTERNAL_AUTH_ID="${CODER_EXTERNAL_AUTH_ID}"
+GITHUB_API_URL="${GITHUB_API_URL}"
+
+if [ -z "$CODER_ACCESS_URL" ]; then
+  echo "No coder access url specified!"
+  exit 1
+fi
+
+if [ -z "$CODER_OWNER_SESSION_TOKEN" ]; then
+  echo "No coder owner session token specified!"
+  exit 1
+fi
+
+if [ -z "$CODER_EXTERNAL_AUTH_ID" ]; then
+  echo "No GitHub external auth id specified!"
+  exit 1
+fi
+
+if [ -z "$GITHUB_API_URL" ]; then
+  echo "No GitHub API URL specified!"
+  exit 1
+fi
+
+echo "Fetching GitHub token..."
+GITHUB_TOKEN=$(coder external-auth access-token $CODER_EXTERNAL_AUTH_ID)
+if [ $? -ne 0 ]; then
+  echo "Failed to fetch GitHub token!"
```

Comment:
The output in this scenario will actually be the URL.

See: https://coder.com/docs/v2/latest/cli/external-auth_access-token#description

It'd be nice to output similarly to slackme in this scenario:

https://github.com/coder/modules/blob/main/slackme/slackme.sh#L64